openapi/resource-server.yaml

```diff
@@ -103,42 +103,6 @@ paths:
       operationId: get-wallet-addres-keys
       description: Retrieve the public keys of the Wallet Address.
       security: []
-  '/connections/{id}':
```

Comment:
is this via the `fetch-schemas.sh`?

Reply:
I think when this commit was originally written Omer modified this file by hand. Could remove this change and commit `fetch-schemas.sh` changes separately.